certified-connectors/Mural/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1074 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MURAL",
+    "description": "Connect MURAL with your favorite apps, so you can automate your work and have more time for what matters most—no code required.",
+    "version": "1.0",
+    "contact": {
+      "name": "MURAL Support",
+      "url": "https://www.mural.co/support",
+      "email": "support@mural.co"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.mural.co/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.mural.co/terms/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Collaboration;Productivity"
+    }
+  ],
+  "host": "app.mural.co",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [],
+  "paths": {
+    "/api/public/v1/murals": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "object",
+                  "properties": {
+                    "backgroundColor": {
+                      "type": "string",
+                      "description": "backgroundColor"
```

Comment:
@dmytroIvanovDev All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)